### PR TITLE
Updated changelog for 10.12.0

### PR DESCRIPTION
**Pull request by zezha-msft**

### CODE REVIEW

File: ChangeLog.md [hunk 1]
```diff
@@ -3,6 +3,25 @@
 
 ## Version 10.11.0
```
Comment (adreed-msft) [inline ChangeLog.md:4]: 10.12.0*

File: ChangeLog.md [hunk 2]
```diff
@@ -3,6 +3,25 @@
...
 
+### New features
+1. Added support for include and exclude regex flags, which allow pattern matching on the entire paths.
+2. Added dry run mode for copy, remove, and sync. This feature allows the user to visualize the changes before committing them.
+3. For SMB aware locations, preserve-smb-info flag is now true by default.
+4. Improved how folder lmts are obtained to allow time-based filters for folders.
+5. Added support for ACL copying between HNS enabled accounts.
```
Comment (adreed-msft) [inline ChangeLog.md:11]: Add note that `preserve-smb-permissions` is now `preserve-permissions`, and the old flag is deprecated (but usable).

File: ChangeLog.md [hunk 3]
```diff
@@ -3,6 +3,25 @@
 
 ## Version 10.11.0
 
+### New features
+1. Added support for include and exclude regex flags, which allow pattern matching on the entire paths.
+2. Added dry run mode for copy, remove, and sync. This feature allows the user to visualize the changes before committing them.
+3. For SMB aware locations, preserve-smb-info flag is now true by default.
+4. Improved how folder lmts are obtained to allow time-based filters for folders.
```
Comment (adreed-msft) [inline ChangeLog.md:10]: I _want_ to call this a bugfix, but I _want_ to call this a feature. a featugfix.

File: ChangeLog.md [hunk 4]
```diff
@@ -3,6 +3,25 @@
...
+5. Added support for ACL copying between HNS enabled accounts.
+
+### Bug fixes
+1. Allow from-to to be set for the remove command.
```
Comment (adreed-msft) [inline ChangeLog.md:14]: In the future, we should investigate more potential issues regarding inconsistencies of flags.

**Review by adreed-msft — CHANGES_REQUESTED** [review]

**Comment by adreed-msft** [comment]
Otherwise approved.

File: ChangeLog.md (hunk 3 above)
Comment (zezha-msft) [reply ChangeLog.md:10]: yah.. it can spin both ways. It's new capability, so let's call it a feature

**Pull request merged**